lusSTR/scripts/filter_settings.py

```diff
@@ -531,6 +531,4 @@ def same_size_filter(df, metadata, datatype):
         else:
             final_df = pd.concat([final_df, df_filt])
     final_df = final_df.reset_index(drop=True)
-    if datatype == "lusplus":
-        final_df = final_df.drop("CE_Allele", axis=1)
```

Comment:
Ran into a problem with this code because the marker plots use the CE allele for plotting- removing it so it can actually create the plots.